KarhooUISDK/Controllers/NavigationController.swift

```diff
@@ -23,6 +64,34 @@ final class NavigationController: UINavigationController {
     private func setupDelegate() {
         delegate = self
     }
+
+    private func setupDesign() {
+        if #available(iOS 13.0, *) {
```

Comment:
maybe merge the bump to ios 13 into this branch to cleanup the availability check?